chapters/methodology/methodology.tex

```diff
@@ -5,8 +5,10 @@ \chapter{Methodology}
 deep-penetration radiation transport problems. In general, the variance reduction
 parameters generated by CADIS and FW-CADIS are not sufficient for
 problems that are strongly
-anisotropic with respect to the flux. The method will extend existing
-methods to generate importance maps--that in turn generate variance reduction
+anisotropic with respect to the flux. The new method will extend existing
```

Comment:
I would say that what we are making is a different type of importance map than what CADIS and FW-CADIS do on their own. I'm hesitant to say that we're making adjoint scalar fluxes (without some quantifier to say it's different than the traditional adjoint) because we aren't strictly an adjoint solution; we require a forward calculation to calculate our modified adjoint, which could be misleading to a novice and/or somebody not as familiar with what we're doing. Nobody can argue that we're making an importance map, because that's absolutely what our flux map is, but they could argue that we're not making a "traditional" adjoint scalar flux.